### DISCUSSION TITLE
Anyone managed to integrate prefetch on hover? Like in turbofetch

### DISCUSSION THREAD

**Discussion by AlvesJorge** [skryukov/inertia_rails-contrib#2] [General]
Heyo! 
Thank you so much for helping popularize Inertia + Rails! It is great to have a company like EvilMartians give it's seal of approval to this combo, and I can't wait for this to take off!

With that said, one of the features I lost when switching over from creating vue app manually and mounting the components through a `*.html.erb` file is link prefetching on hover.

If I still include `import "@hotwired/turbo-rails"` on my `application.js` file I see that it indeed still does the prefetching, but inertia doesn't use it and instead fetches the document again.

Has anyone found to integrate resource pre-fetching on Link/Anchor tag hover? Doesn't have to be with turbo :)

**Comment by skryukov** [comment]
Hey, @AlvesJorge, thanks for the kind words!

To my knowledge, no, there is no prefetching available for Inertia. There is a closed PR https://github.com/inertiajs/inertia/pull/223/files which we can probably revisit and release as an add-on 🤔 Right now I'm working on extended TS support, so I'll be able to look deeply into it a bit later.
